src/main/java/com/sintern/api/CompanyController.java

```diff
@@ -47,4 +50,13 @@ public void uploadLogo(@RequestParam("id") UUID companyId, @RequestParam("image"
     public GetCompanyResponse getCompany(@PathVariable UUID companyId){
         return GetCompanyTransformer.transform(companyService.findByID(companyId));
     }
+
+    @PostMapping("/update")
```

Comment:
Not wrong, but when updating an Entity, we use the **PUT HTTP method**, without specifying any additional path. For instance, the root endpoint is **"/companies"** and **HTTP method** is **PUT** means that we're updating a **Company**. It would have been clearer if the API had been **"/companies/{companyId}"**, but it's also fine the way it is.

Please replace the **HTTP Method** to **PUT**, without specifying any additional path.